clang/test/CodeGenSYCL/simplifycfg.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %clangxx -fsycl -fsycl-device-only %s -O3 -S -o - | FileCheck %s
+//
+// Test that SimplifyCFG pass isn't invoked by checking number of function's calls.
```

Comment:
This test checks that `_Z32__spirv_SubgroupShuffleDownINTELIiET_S0_S0_j` is called twice after O3 optimizations.